From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Mon, 6 Jul 2015 09:59:30 -0300
Subject: [PATCH] build: add libavltree, libcommon & libheap dependencies

Otherwise it can break on very parallel builds since collectd link time
arrives before one or more of these were built.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
---
 src/daemon/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/daemon/Makefile.am b/src/daemon/Makefile.am
index 00a8fd7..99dc158 100644
--- a/src/daemon/Makefile.am
+++ b/src/daemon/Makefile.am
@@ -73,7 +73,7 @@ collectd_CPPFLAGS =  $(AM_CPPFLAGS) $(LTDLINCL)
 collectd_CFLAGS = $(AM_CFLAGS)
 collectd_LDFLAGS = -export-dynamic
 collectd_LDADD = libavltree.la libcommon.la libheap.la -lm
-collectd_DEPENDENCIES =
+collectd_DEPENDENCIES = libavltree.la libcommon.la libheap.la
 
 # The daemon needs to call sg_init, so we need to link it against libstatgrab,
 # too. -octo
